halo2_gadgets/src/ecc/chip/mul_fixed/short.rs

```diff
@@ -580,7 +580,7 @@ pub mod tests {
                                     offset: 1,
                                 },
                                 cell_values: vec![(
-                                    ((Any::Advice, 5).into(), 0).into(),
+                                    ((Any::Advice { phase: First }, 5).into(), 0).into(),
```

Comment:
These would become something like:
```suggestion
                                    ((VirtualColumn::Advice(First), 5).into(), 0).into(),
```